src/main/scala/latis/input/HapiAdapter.scala

```diff
@@ -81,21 +83,21 @@ abstract class HapiAdapter(model: DataType, config: HapiAdapter.Config) extends
 
     // Updates query info based on each Operation
     ops.foreach {
-      case Selection("time", op, value) =>
+      case Selection(id, op, value) if id == id"time" =>
         val time = TimeFormat.parseIso(value).getOrElse {
           val msg = s"Failed to parse time: $value"
           throw LatisException(msg)
         } //ms since 1970
-        op.head match {
-          case '>' =>
+        op match { //TODO: support GtEq and LtEq?
```

Comment:
I added this TODO because it surprised me that we're only supporting `Gt`, `Lt`, and `Eq`. I noticed this because a unit test was failing with `"Unsupported select operation: >="`. The test was doing a 
```scala
Selection("time >= 2010")
```
and broke when I changed it to the updated syntax:
```scala
Selection(id"time", GtEq, "2010"))
```

I assume the test used to pass because the selection was being parsed outside of this pattern match until I used the updated syntax. Is there any reason we should **not** add support for at least `GtEq` and `LtEq`?

Reply:
I'm guessing this was just matching the first character of the selection operator string (`op.head`), so `>` would have matched both `>` and `>=`.

If that's the case, then you should be able to replicate that behavior with:

```scala
case Gt | GtEq => ???
case Lt | LtEq => ???
```

Reply:
Cool. Done. 👍